EssentialsProtect/src/com/earth2me/essentials/protect/ProtectConfig.java

```diff
@@ -26,18 +27,23 @@
     prevent_block_on_rail("protect.protect.prevent-block-on-rails", false),
     prevent_tnt_explosion("protect.prevent.tnt-explosion", false),
     prevent_tnt_playerdmg("protect.prevent.tnt-playerdamage", false),
+    prevent_tnt_itemdmg("protect.prevent.tnt-itemdamage", false),
```

Comment:
Is there any chance you could add the new options to the default config?

Reply:
> Is there any chance you could add the new options to the default config?

Sorry, I don't understand what you are requesting. What do you mean by default config?

Reply:
> Is there any chance you could add the new options to the default config?

no problem, done in the latest commit 👍